di-ipv-evcs-stub/lambdas/src/domain/vcsResponse.ts

```diff
@@ -0,0 +1,6 @@
+import VcStateMetadata from "./vcStateMetadata";
+
+export default interface VcsResponse {
+  vcs: VcStateMetadata[];
+  afterKey: string;
+}
```

Comment:
Looks like this currently isn't being used anywhere?

Reply:
removed